### PR TITLE
Object.xml -- updating samples for Try .NET

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
docs PR is https://github.com/dotnet/dotnet-api-docs/pull/3346

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equalssb1.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
-﻿// <Snippet5>
-using System;
+﻿using System;
 using System.Text;
 
 public class Example
 {
    public static void Main()
    {
+      // <Snippet5>
```
Comment (mairaw) [inline snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equalssb1.cs]: please undo this change. This one required System.Text so it doesn't work on try.net as-is

Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equalssb1.cs]: Reverted.
